<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ZenScript Code Generator by HLNikNiky</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Text</family>
     <pointsize>21</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ZenScript Code Generator</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Assembler</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Text</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Items from CT entry</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>add</string>
      </attribute>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>80</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine11">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine7">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox9">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox8">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox10">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox11">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine9">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox7">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>0</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine12">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="assemblerSpinBox12">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLine8">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Item Output:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="assemblerLineOutput">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>120</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Int duration</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="intDurationSpinBox">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>remove</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove item</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="saveRemoveButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>replace</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hbm Version</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
